### PR TITLE
Add instruction to get secret config for project setup

### PR DESCRIPTION
**Pull request by SteveJacksonSoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,27 @@ In JetBrains Rider the text editor settings can be set in `Settings > Editor > F
...
+Talk to the project team to get these secrets for you to store in the secrets manager.
+
+## Secrets manager
+We use Microsoft’s [Secret Manager](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows#secret-manager) to look after our secrets locally, so they don’t get tracked (or deleted) by git.
+
+It stores a GUID in the Web project’s csproj which points .NET to a json file in your appsettings when you run the application.
+
+Namely: `~\AppData\Roaming\Microsoft\UserSecrets\7ea176d2-09f5-4e3a-a3fa-7f3229882b70\secrets.json`
+
+## Environment variables
```
Comment (SteveJacksonSoft) [inline README.md:51]: Shifted this section from the bottom of the readme

File: README.md [hunk 2]
```diff
@@ -36,6 +36,27 @@ In JetBrains Rider the text editor settings can be set in `Settings > Editor > F
 
 To turn off git autoCrlf, run `git config --global core.autocrlf false` in a terminal.
 
+# Secret config settings
+Certain features of the system rely on having API keys and secrets available (as do some tests).
+
+Talk to the project team to get these secrets for you to store in the secrets manager.
+
+## Secrets manager
```
Comment (SteveJacksonSoft) [inline README.md:44]: Copied this straight from https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19902234654/Secret+config+settings

File: README.md [hunk 3]
```diff
@@ -36,6 +36,27 @@ In JetBrains Rider the text editor settings can be set in `Settings > Editor > F
 
 To turn off git autoCrlf, run `git config --global core.autocrlf false` in a terminal.
 
+# Secret config settings
+Certain features of the system rely on having API keys and secrets available (as do some tests).
+
+Talk to the project team to get these secrets for you to store in the secrets manager.
```
Comment (SteveJacksonSoft) [inline README.md:42]: This is the only new stuff

**Review by AlexJacksonDS — COMMENTED** [review]
New stuff looks fine. While your fixing it can you fix the command under Getting the code. It currently says `git checkout https://github.com/TechnologyEnhancedLearning/DLSV2.git`

**Comment by SteveJacksonSoft** [comment]
> New stuff looks fine. While your fixing it can you fix the command under Getting the code. It currently says `git checkout https://github.com/TechnologyEnhancedLearning/DLSV2.git`

Think Simon's going to fix that along with some other stuff when he gets access to github.

**Pull request merged**